lib/mongoose_push/service/fcm/pool_supervisor.ex

```diff
@@ -0,0 +1,49 @@
+defmodule MongoosePush.Service.FCM.Pool.Supervisor do
```

Comment:
The name of this file doesn't match the module name. It should be ` lib/mongoose_push/service/fcm/pool/supervisor.ex`.